src/Conf.ml

```diff
@@ -1305,6 +1305,18 @@ let default_profile =
   ; wrap_comments= C.default Formatting.wrap_comments
   ; wrap_fun_args= C.default Formatting.wrap_fun_args }
 
+let conventional_profile =
+  { default_profile with
+    break_infix_before_func= false
+  ; break_separators= `After_and_docked
+  ; break_sequences= true
+  ; field_space= `Loose
+  ; indicate_nested_or_patterns= false
+  ; sequence_style= `Terminator
+  ; space_around_collection_expressions= true }
```

Comment:
This option does not work on type declaration combined with `break-separators=after-and-docked`:

```ocaml
type t = {x : int}
```

With `break-separators=after`:

```ocaml
type t = { x : int }
```

Reply:
https://github.com/ocaml-ppx/ocamlformat/pull/670